datalad_deprecated/resources/website/assets/js/main.js

```diff
@@ -594,4 +732,4 @@ function clearCache() {
 }
 
 window.addEventListener('beforeunload', clearCache, false);
-*/
+*/
```

Comment:
please don't delete trailing new line here -- unrelated to the changes in PR and in general "good to have"